README.md

```diff
@@ -51,6 +54,34 @@ class Task extends AbstractScheduledTask {
 }
 ```
 
-Your task is now scheduled and will be called every 5 minutes.
+Your task is now scheduled and will be called every 5 minutes.  
+
+### Every X Hours or Daily
+
+if you are scheduling your task on an hourly or daily basis, you'll need to set `minutes()` in addition to `hours()`, `everyHours()`, or `daily()`.
```

Comment:
Can you add more details about what is happening when this is **not** used? Like: "If you omit calling `minutes`, the task is run each minute"?